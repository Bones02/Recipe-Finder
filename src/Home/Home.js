import React, { Component } from "react";
import {Link} from 'react-router-dom';
import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
import './Home.css';


class Home extends Component {
  render() {
    return (
      <div className="home">
        <h1>Welcome to the Recipe Finder App</h1>
        <p>Recipe Finder helps you search for recipes by 
          ingredient and save your favorites for later! 
          You can access all your favorite recipes by heading 
          over to the My Recipes page.</p>

        <h2>
            <Link to="/App">Try it out for yourself!</Link>{' '}
            <FontAwesomeIcon icon= "coffee"/>
        </h2>

      </div>
    );
  }
}
 
export default Home;