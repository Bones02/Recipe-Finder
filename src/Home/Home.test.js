import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Home from './Home'

describe(`Home component`, () => {
  it('renders a .home by default', () => {
    const wrapper = shallow(<Home/>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})