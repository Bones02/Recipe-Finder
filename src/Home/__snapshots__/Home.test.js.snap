// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders a .home by default 1`] = `
<div
  className="home"
>
  <h1>
    Welcome to the Recipe Finder App
  </h1>
  <p>
    Recipe Finder helps you search for recipes by ingredient and save your favorites for later! You can access all your favorite recipes by heading over to the My Recipes page.
  </p>
  <h2>
    <Link
      replace={false}
      to="/Search"
    >
      Try it out for yourself!
    </Link>
     
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon="coffee"
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size={null}
      spin={false}
      swapOpacity={false}
      symbol={false}
      title=""
      transform={null}
    />
  </h2>
</div>
`;
